statick_tool/plugins/tool/make_tool_plugin.py

```diff
@@ -101,10 +101,13 @@ def parse_output(self, package, output):
         matches = self.filter_matches(matches, package)
         issues = []
         for match in matches:
+            cert_reference = None
             warning_list = warning_parse.match(match[4])
+            if warning_list is not None and warning_list.groups(1)[0] in warnings_mapping.keys():
+                cert_reference = warnings_mapping[warning_list.groups(1)[0]]
 
             if warning_list is None:
-                # Something's gone wrong if we don't match the [warning] format
+                # Something's gone wrong if we don't match the [waring] format
```

Comment:
This just added a typo.

Reply:
So it did. Not sure where that came from, will fix.